### ISSUE TITLE
Can the file name be removed from the bottom left of the screen?

### ISSUE THREAD

**Issue by JohnnieBBass** [mvysny/photocloud-frame-slideshow#154]
Thank you for making your time and reporting the issue. I can't possibly test on all devices,
so it is easy for me to overlook bugs which may cause PhotoCloud to be unusable on your
device. Also, everybody will benefit from the fixed bug or a new feature.

Since the Google Play revenue is just peanuts, please consider posting a bounty on the
bug. Posting a bounty drastically reduces the fix time.

PhotoCloud includes an automatized bug reporting system, so it is possible that I
already have a bug report and I'm working on a fix. Yet, still please feel free to
create a bug and attach a screenshot of the bug, and steps which lead to the bug.

Please, if the issue is fixed for you, I urge you to change the rating. One-star
ratings discourage new users from using PhotoCloud. That means less bugs reported and
fixed, less features asked and implemented, and I get paid less which discourages
me from implementing more features. Thanks!

**Comment by mvysny** [comment]
Yes, head to Settings / Widgets/HUD / and disable "Show EXIF in Slideshow" and "Show Clock"
